Document "Zigbee Groups" in ZHA integration (#29316)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

In practice, you will likely need to add a lot more Zigbee router devices than in this example in order to extend the coverage of the network to reach that many devices.

## Zigbee Groups, Bindings and Unbindings

ZHA integration support for native Zigbee Groups, as well as Zigbee Bindings and Unbindings. These features can be used separately or combined.

### Zigbee Group

A "Zigbee Group" enables native on-device grouping of multiple Zigbee lights, switches, and fans that enable controlling all compatible entities for those devices in those groups with only one command/entity.

> Importantly note that using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration should make for much faster response times and better synchronization, especially when grouping lights.

Reviewer: I think the "importantly" sound a bit off. Maybe change the phrasing to something like "Note that..." and put the whole paragraph into a callout block (if it's really that important).

```suggestion
<div class='note'>
Note that using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration should make for much faster response times and better synchronization, especially when grouping lights.
</div>
```

To create a Zigbee Group click on "CONFIGURE" in the Zigbee Home Automation integration, at the top of the screen you will find "Groups", and there you can select "+ CREATE GROUP". Give the group a name and choose which devices that are compatible with each other should be in that group.

The group should consist of products of the same device type, (e.g. all lights, switches, or fans), and at least two entities must be added to a Zigbee group inside the ZHA integration before a group entity is created.

### Zigbee Binding and Unbinding

> Binding is a native action in Zigbee which defines on-device relations between two Zigbee devices, specific endpoints, and cluster id. It provides a mechanism for attaching an endpoint on one Zigbee node to one or more endpoints on another Zigbee node or Zigbee group (a group of Zigbee devices).

Reviewer: The second sentence is redundant ("Zigbee group (a group of Zigbee devices)"), isn't is?

```suggestion
Binding is a native action in Zigbee which defines on-device relations between two Zigbee devices, specific endpoints, and cluster id. It provides a mechanism for attaching an endpoint on one Zigbee node to one or more endpoints on other Zigbee nodes or groups.
```

In practice, you will likely need to add a lot more Zigbee router devices than in this example in order to extend the coverage of the network to reach that many devices.

## Zigbee Groups, Bindings and Unbindings

ZHA integration support for native Zigbee Groups, as well as Zigbee Bindings and Unbindings. These features can be used separately or combined.

### Zigbee Group

A "Zigbee Group" enables native on-device grouping of multiple Zigbee lights, switches, and fans that enable controlling all compatible entities for those devices in those groups with only one command/entity.

<div class='note'>

> Note that using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration should make for much faster response times and better synchronization, especially when grouping lights.

Reviewer 2: This is actually something that isn't intuitive and may actually be harmful: using Zigbee groups improves the visual responsiveness (as in, devices will toggle at the same time) but it is only recommended if you control *any* Zigbee groups at most once or twice a minute, as Zigbee group commands are broadcasts and can only be sent every few seconds. Any more often will cause the group command to be rejected by the firmware.

Author: Addedinformation about that.

In practice, you will likely need to add a lot more Zigbee router devices than in this example in order to extend the coverage of the network to reach that many devices.

> ## Zigbee Groups, Bindings and Unbindings

Reviewer 3: ```suggestion
## Zigbee groups and binding devices
```

> ZHA integration support for native Zigbee Groups, as well as Zigbee Bindings and Unbindings. These features can be used separately or combined. Binding a remote directly to a bulb or group has the benefit of faster response time and smoother control. This can improve user visual experience like dimming lights as the remote then directly dims the light and thus does not have to make the software roundtrip via the ZHA coordinator.

Reviewer 3: ```suggestion
ZHA supports Zigbee groups and binding devices to each other. These features can be used separately or combined. For example, binding a remote to a bulb or group has the benefit of faster response time and smoother control, as the remote directly controls the bound devices.
```

> ### Zigbee Group

Reviewer 3: ```suggestion
### Zigbee group
```

> A "Zigbee Group" enables native on-device grouping of multiple Zigbee lights, switches, and fans that enable controlling all compatible entities for those devices in those groups with only one command/entity.

Reviewer 3: ```suggestion
A Zigbee group enables the grouping of multiple Zigbee lights, switches, and fans. This allows you to control those devices with only one command/entity.
```

<div class='note'>

> Note that while using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration can improve the visual responsiveness and better synchronization grouping lights, the control commands broadcast can be rejected by devices if try to issue it to any Zigbee groups more than once or twice per minute.

Reviewer 3: ```suggestion
Note that while using a native Zigbee group instead of Home Assistant's [Group](/integrations/group/) integration can improve the visual responsiveness, the broadcast commands issued can flood the Zigbee network if issued repeatedly.
```

</div>

> To create a Zigbee Group click on "CONFIGURE" in the Zigbee Home Automation integration, at the top of the screen you will find "Groups", and there you can select "+ CREATE GROUP". Give the group a name and choose which devices that are compatible with each other should be in that group.

Reviewer 3: ```suggestion
To create a Zigbee Group, press the "Configure" button on the ZHA integration config page. At the top, choose "Groups" and select "Create Group". Set a group name and choose which devices to include in the group.
```

> The group should consist of products of the same device type, (e.g. all lights, switches, or fans), and at least two entities must be added to a Zigbee group inside the ZHA integration before a group entity is created.

Reviewer 3: ```suggestion
The group should consist of products of the same device type (e.g. all lights, switches, or fans), and at least two devices must be added to a Zigbee group before a group entity is created.
```

> ### Zigbee Binding and Unbinding

Reviewer 3: ```suggestion
### Zigbee binding and unbinding
```
(not sure about which capitalization we want here, but we generally seem to use what's used in a normal sentence(?))

Binding is an on-device feature for Zigbee devices. It provides a mechanism for attaching an endpoint of one Zigbee device to an endpoint of another Zigbee device or to a Zigbee group.

> For example, binding a "target destination" Zigbee device like a remote to a Zigbee light bulb, switch or group of light bulbs allows direct control of the "target" device (light, switch, shade) from the "remote" Zigbee device, bypassing ZHA. This means that the remote can control the light bulb or group even when ZHA is not active.```

Reviewer 2: Extraneous ` ``` `:

```suggestion
For example, binding a "target destination" Zigbee device like a remote to a Zigbee light bulb, switch or group of light bulbs allows direct control of the "target" device (light, switch, shade) from the "remote" Zigbee device, bypassing ZHA. This means that the remote can control the light bulb or group even when ZHA is not active.
```

> Binding is a native on-device action for Zigbee devices, defining relations between specific endpoints two Zigbee devices. It provides a mechanism for attaching an endpoint on one Zigbee device to one or more endpoints other Zigbee devices or a Zigbee group.

Reviewer 3: ```suggestion
Binding is an on-device feature for Zigbee devices. It provides a mechanism for attaching an endpoint of one Zigbee device to an endpoint of another Zigbee device or to a Zigbee group.
```

> Note that not all devices support binding. By default, ZHA binds remotes to the coordinator, so click events are forwarded to HA. As some remotes can only be bound to a single destination, you might need to unbind the remote from the coordinator before binding it to another device or group.

Author: That is out of scope for this PR. Someone else can add auch details later if still wanted.

> For example, binding a "target destination" Zigbee device like a remote to a Zigbee lightbulb, switch or group of lightbulbs allows direct control of the "target" device (light, switch, shade) from the "remote" Zigbee device, bypassing ZHA.  This means that the remote can control the lightbulb/group of lightbulbs even when the Zigbee coordinator is not available.

Reviewer 3: ```suggestion
For example, binding a "target destination" Zigbee device like a remote to a Zigbee light bulb, switch or group of light bulbs allows direct control of the "target" device (light, switch, shade) from the "remote" Zigbee device, bypassing ZHA. This means that the remote can control the light bulb or group even when ZHA is not active.```

> Note that not all devices support binding as it depends on the Zigbee implementation of the device itself. Also, by default, the ZHA integration binds remotes to the coordinator when joining/pairing them, so the coordinator can receive commands from the remotes and originate zha_events. However, some remotes can only be bound to a single destination and it is not possible to make this switch to bind to other destinations like a device or groups unless you first unbind the remote from the coordinator. Only after you unbind such a remote from the ZHA coordinator you can then bind it directly to any other Zigbee device or a group.

Reviewer 3: ```suggestion
Note that not all devices support binding. By default, ZHA binds remotes to the coordinator, so click events are forwarded to HA. As some remotes can only be bound to a single destination, you might need to unbind the remote from the coordinator before binding it to another device or group.
```
